common/app/views/support/cleaner/amp_embed_cleaner/AmpEmbed.scala

```diff
@@ -0,0 +1,11 @@
+package views.support.cleaner.amp_embed_cleaner
```

Comment:
I would prefer `views.support.cleaner.amp.AmpEmbed`.  That way the other AMP cleaner can go in the `amp` package too (and we avoid the underscores in package names).